lib/Graph/Nodes.cpp

```diff
@@ -1022,6 +1022,32 @@ bool FusedRowwiseQuantizedSparseLengthsWeightedSumNode::verify() const {
   return isValid;
 }
 
+bool FusedRowwiseQuantizedSparseLengthsSumNode::verify() const {
+  bool isValid = checkType(getResult(), ElemKind::FloatTy, this);
```

Comment:
Since this is virtually all the same logic from `FusedRowwiseQuantizedSparseLengthsWeightedSumNode::verify()`, can you outline it and share it mostly? E.g. take a look at what we do with `verifyConvolution`. You could pass an empty `NodeValue` for `Weights` and only verify things related to it when not empty.

Reply:
Done.